lua/autorun/server/sv_acf_legal_checks.lua

```diff
@@ -18,7 +18,26 @@ local legalChecks = {
     end
 }
 
+local badMaterials = {
+    ["models/effects/vol_light001"] = true,
+    ["models/effects/comball_tape"] = true,
+    ["models/props_combine/portalball001_sheet"] = true,
+    ["models/effects/comball_sphere"] = true
+}
+
 local function checkLegal( ent )
+    -- Checks for all ent types
+    local color = ent:GetColor()
+    if color.a <= 10 then
+        return false, "Low Alpha", "Your acf part has a low alpha value and has been disabled."
+    end
+
+    local mat = string.lower( ent:GetMaterial() )
+    if badMaterials[mat] then
+        return false, "Bad Material", "Your acf part has a bad material and has been disabled."
```

Comment:
```suggestion
        return false, "Bad Material", "Your ACF part has a bad material and has been disabled."
```

Would it make sense to include the material name in the error message?

Reply:
![image](https://user-images.githubusercontent.com/69946827/200047905-ca5ac5c5-add7-4a42-87b6-449642b314de.png)